src/Symfony/Component/Validator/Tests/Mapping/Loader/AnnotationLoaderTest.php

```diff
@@ -185,5 +185,9 @@ public function provideNamespaces(): iterable
         if (\PHP_VERSION_ID >= 80000) {
             yield 'attributes' => ['Symfony\Component\Validator\Tests\Fixtures\Attribute'];
         }
+
+        if (\PHP_VERSION_ID >= 80100) {
+            yield 'nested_attributes' => ['Symfony\Component\Validator\Tests\Fixtures\Attribute\Nested'];
```

Comment:
```suggestion
            yield 'nested_attributes' => ['Symfony\Component\Validator\Tests\Fixtures\NestedAttribute'];
```

Let's keep the directory structure flat

Reply:
Alright!